src/index.js

```diff
@@ -67,16 +67,22 @@ exports.handler = async (event, _context, callback) => {
     const optimizelyClient = createInstance({
       datafile,
       logLevel: OptimizelyEnums.LOG_LEVEL.ERROR,
-      clientEnginer: AWS_LAMBDA_AT_EDGE_CLIENT_ENGINE,
+      clientEngine: AWS_LAMBDA_AT_EDGE_CLIENT_ENGINE,
       /**
        * You can add other Optimizely SDK initialization options here, such as a custom event dispatcher
        * for sending impression events to Optimizely LogX.
-       *
-       * Note: The event dispatcher below is a sample implementation of fire-and-forget event dispatching
+       */
+
+      /**
+       * Optional event dispatcher. Please uncomment the following line if you want to dispatch an impression event to optimizely logx backend.
+       * When enabled, an event is dispatched asynchronously. It does not impact the response time for a particular worker but it may
+       * add to the total compute time of the Lambda function and can impact AWS billing.
+       * 
+       * The event dispatcher attached below is a sample implementation of fire-and-forget event dispatching
        * in Lambda, however if your needs are more complex, you can implement your own event dispatcher
        * and integrate with Step Functions, SQS, or other services.
        */
-      dispatchEvent: dispatchEvent,
+      // eventDispatcher: dispatchEvent,
```

Comment:
dispatchEvent is a function in the eventDispatcher object / instance so it will go like
```
eventDispatcher: { dispatchEvent },
```